pkg/config/setting.go

```diff
@@ -522,13 +521,12 @@ func GenPodAttrWithMountPod(ctx context.Context, client *k8sclient.K8sClient, mo
 	}
 	attr.Resources = resources
 	setting := &JfsSetting{
-		IsCe:      IsCEMountPod(mountPod),
-		PV:        pv,
-		PVC:       pvc,
-		Name:      mountPod.Annotations[JuiceFSUUID],
-		VolumeId:  mountPod.Annotations[UniqueId],
-		MountPath: filepath.Join(PodMountBase, pvName) + mountPod.Name[len(mountPod.Name)-7:],
```

Comment:
we need this to parse config template

Reply:
the method of get mount pod is in util/resource, and it will be circular refered here. will raise another pr to fix it.